charts/tfy-gpu-operator/templates/aks-nvidia-device-plugin.yaml

```diff
@@ -50,17 +50,20 @@ spec:
           value: spot
           effect: NoSchedule
       containers:
-        - image: mcr.microsoft.com/oss/nvidia/k8s-device-plugin:1.11
+        - image: nvcr.io/nvidia/k8s-device-plugin:v0.14.1
           name: nvidia-device-plugin-ctr
+          env:
+            - name: FAIL_ON_INIT_ERROR
```

Comment:
Is there a reason for this?

Reply:
I followed the recommendation in the linked comment in description. Actually now you mention it, let me remove it,
Fail on init false is required to run the daemonset on nodes that don't have gpus which we have avoided via affinity anyway

Thanks for bringing this up